libs/blocks/section-metadata/sticky-section.js

```diff
@@ -17,7 +18,8 @@ function promoIntersectObserve(el, stickySectionEl, options = {}) {
       const isPromoStart = entry.target === stickySectionEl;
       const abovePromoStart = (isPromoStart && entry.isIntersecting)
         || stickySectionEl?.getBoundingClientRect().y > 0;
-      if (entry.isIntersecting || abovePromoStart) el.classList.add('hide-sticky-section');
+      if (entry.target === document.querySelector('footer')) isFooterStart = entry.isIntersecting;
```

Comment:
Do we really need the conditional?
```suggestion
      isFooterStart = entry.target === document.querySelector('footer') && entry.isIntersecting;
```

Reply:
It might seem unnecessary, but when the observer callback is triggered by another observed element, we do not want to change the value of `isFooterStart`. Removing the `if statement` could lead to scenarios where the footer is visible, but `isFooterStart` is set to false.